R/utils.R

```diff
@@ -34,20 +34,94 @@ NULL
 
 
 #' @title
-#' Count missing
+#' Number of missingness
 #'
 #' @description
-#' This function checks the number of missingness in a vector, returns the results in the exact number (percentage)
+#' This function checks number of missingness in a `vector`, or a `data.frame`
+#' or `data table`. It returns the results in exact number (percentage) or
+#' returns the index of missingness
 #'
-#' @param x (`vector`)
+#' @param x (`vector` or `data.frame` or `data.table`)
+#' the object to be checked
+#'
+#' @param na_strings (`character`)
+#' a character vector of strings which are to be interpreted as `NA` values.
+#' The default for `n_missing` is "", which will treat empty strings as `NA`.
+#'
+#' @param index (`logical`)
+#' If true the function returns the index of the missing values instead of the
+#' number (percentage). For `vector` input, a logical vector of same length will
+#' be returned. For `data.frame` or `data.table` input, the function returns a
+#' vector with length equal to the number of rows of input table and any
+#' missingness in the row will result in a returned value of `TRUE` for the row.
+#'
+#' @return (`logical`)
+#' For `vector` input, a vector of logical values the same length of `x` where
+#' each value represents whether that particular element is missing.
+#' For `data.frame` or `data.table` input, a vector of logical values where
+#' length equals to number of rows of input table and each value represents
+#' whether any element of the row is missing.
```

Comment:
I'm not sure if this is true? Isn't this outputting a named `character` vector and not a `logical` vector?

Reply:
It should be `character` or `logical` depending on the input `index` argument. `character` is missed here and I will add it.